server/routes/creatingLicences/handlers/additionalLicenceConditionsCallback.ts

```diff
@@ -36,4 +36,10 @@ export default class AdditionalLicenceConditionsCallbackRoutes {
     }
     return res.redirect(`/licence/create/id/${licenceId}/bespoke-conditions-question`)
   }
+
+  needsReview = (condition: AdditionalCondition) => {
+    return condition.data.some(data => {
+      return data.field === 'infoInputReviewed' && data.value !== 'true'
+    })
```

Comment:
nice to have some test coverage for this?

Reply:
Sorry, I didn't mean a unit test for this function - more a supertest test to cover how this impacts navigation